web-ui/src/components/member-directory/AdminMemberCard.css

```diff
@@ -0,0 +1,46 @@
+.member-card {
```

Comment:
If these are the same as those in the directory component, then they will clash (only one will apply).  I would either remove one of them or change the base `member-card` class to `admin-member-card` or the like